lib/plugins/styling/_test/general.test.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * General tests for the styling plugin
```

Comment:
IMHO it is not very useful. There are other things in this plugin which are much more worth writing tests for.
The plugin wizard also generates the plugin.info.txt, so testing for it makes even less sense.

I can imagine it was only meant as a starting point for tests in general (as that's the only thing you can test in an otherwise yet unknown plugin) from the plugin wizard's perspective.